README.md

```diff
@@ -1 +1,113 @@
-# math-magician
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Math Magician
+
+## Built With
+
+- HTML5
+- CSS
+- JavaScript
+- React
+
+
+### Setup
+
+- Clone this repo URL "https://github.com/vanillainanutshell/math-magician"
+
+## Authors
+
+👤 **Author**
```

Comment:
- [ ] OPTIONAL
Kindly replace the text **Author** with your full name, so as to make your ReadMe more professional